python/ql/test/query-tests/Security/CWE-377/InsecureTemporaryFile.expected

```diff
@@ -0,0 +1 @@
+| InsecureTemporaryFile.py:5:16:5:23 | ControlFlowNode for mktemp() | Call to deprecated function $@.$@ may be insecure. | tempfile | mktemp |
```

Comment:
No results for `os.tempnam` or `os.tmpnam`?

Reply:
Indeed. This is what I've been discussing with you on Slack for the past hour...

Reply:
You shouldn't submit incorrect test results.
The test results should reflect what the query is supposed to do.